chapter-1-passing-tests.zig

```diff
@@ -597,11 +595,7 @@ test "++" {
 test "**" {
     const pattern = [_]u8{ 0xCC, 0xAA };
     const memory = pattern ** 3;
-    expect(eql(
-        u8,
-        &memory,
-        &[_]u8{ 0xCC, 0xAA, 0xCC, 0xAA, 0xCC, 0xAA }
-    ));
+    expect(eql(u8, &memory, &[_]u8{ 0xCC, 0xAA, 0xCC, 0xAA, 0xCC, 0xAA }));
```

Comment:
I like the previous version more (it's more readable, which would be great for zig beginners), so I left this piece of code unchanged in `chapter-1.md`, but I don't think that I should leave it not formatted in the source code, since `zig fmt` will change it eventually.